doc/source/reference/fairseq2.nn.Embedding.rst

```diff
@@ -1,7 +1,7 @@
 .. _embedding:
 
-Embeddings
-==========
+fairseq2.nn.Embedding
```

Comment:
I think this should be in lower-case, since this is the name of the module. `Embedding` (the interface) is on line 8 in this file.